account_banking_payment_export/models/payment_mode.py

```diff
@@ -94,16 +94,27 @@ def _default_type(self):
             or self.env['payment.mode.type']
 
     type = fields.Many2one(
-        'payment.mode.type', string='Export type', required=True,
+        'payment.mode.type',
```

Comment:
Don't make these changes if not needed. It's only your style preference (one argument per line). I prefer for instance the other style, that don't need too many lines for the same. Both are PEP8.